composer.json

```diff
@@ -40,17 +40,18 @@
         "doctrine/collections": "^1.0",
         "doctrine/common": "^2.13 || ^3.0",
         "doctrine/event-manager": "^1.0",
-        "doctrine/persistence": "^1.3.4 || ^2.0"
+        "doctrine/persistence": "^1.3.3 || ^2.0"
     },
     "conflict": {
-        "doctrine/mongodb": "<1.3",
+        "doctrine/cache": "<1.11",
+        "doctrine/dbal": "<2.13.1 || ~3.0.0",
         "doctrine/mongodb-odm": "<2.0",
         "doctrine/orm": "<2.10.2",
         "sebastian/comparator": "<2.0"
     },
     "require-dev": {
         "doctrine/cache": "^1.11 || ^2.0",
-        "doctrine/dbal": "^2.13",
+        "doctrine/dbal": "^2.13.1 || ^3.1",
```

Comment:
2.13.0 had a few bugs with the forward compat layer, and 3.1 had various improvements in the compat layer.  This is the same constraint Symfony is using in the framework, should be good enough here.